math/mean.go

```diff
@@ -0,0 +1,31 @@
+package maths
+
+import (
+	"golang.org/x/exp/constraints"
+)
+
+type Digit interface {
```

Comment:
We already have those [constraints](https://github.com/TheAlgorithms/Go/blob/master/constraints/contraints.go#L28-L30)